packages/administration/templates/content/complaint/list.html.twig

```diff
@@ -35,7 +35,8 @@
                         form_input: quickSearchForm.text,
                         form_submit: quickSearchForm.submit,
                         hide_title: true,
-                        info_message: 'Searching searches in <ul> <li>complaint number</li><li>complaint email</li><li>order number</li><li>delivery last name</li><li>delivery company name</li><li>last name from order</li><li>email from order</li><li>complaint author last name</li><li>complaint author company name</ul> It is possible to use wildcards * and ?'|trans,
+                        placeholder: 'Search by complaint number, email, name or UUID...'|trans,
```

Comment:
it is rather a nitpick, but we should use oxford comma when there are 3 and more items in the list :) https://www.scribbr.co.uk/comma/the-oxford-comma/